added subsystem

cocothebam opened the pull request:
moves backwards and forwards

cocothebam committed:
added subsystem

moves backwords and forwords

KenwoodFox commented on src/main/java/frc/robot/subsystems/Drivetrain.java:
```diff
@@ -0,0 +1,33 @@
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * Add your docs here.
```
Add in some documentation here, this will be seen when people use your subsystem elsewhere in the code.

KenwoodFox commented on src/main/java/frc/robot/subsystems/Drivetrain.java:
```diff
@@ -0,0 +1,33 @@
+ * Add your docs here.
+ */
+public class Drivetrain extends Subsystem {
+public static void JoyStickLever (TalonSRX StarBoard, TalonSRX port, Joystick JoyStickLever) {
```
I recommend making your variables lowercase.

KenwoodFox commented on src/main/java/frc/robot/subsystems/Drivetrain.java:
```diff
@@ -0,0 +1,33 @@
+ */
+public class Drivetrain extends Subsystem {
+public static void JoyStickLever (TalonSRX StarBoard, TalonSRX port, Joystick JoyStickLever) {
+  Double thro = JoyStickLever.getRawAxis(1);
```
Try implementing a yaw feature, you can use the code in joe-drivetrain as an example, the axis for yaw is 2

KenwoodFox reviewed (changes requested):
Awesome work today! Make those code changes when you can.

KenwoodFox commented on src/main/java/frc/robot/subsystems/Drivetrain.java:
```diff
@@ -0,0 +1,33 @@
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.I2C.Port;
```
You don't need this line
`import edu.wpi.first.wpilibj.I2C.Port;`

cocothebam committed:
added values and Robot Telliop

added values and Robot Telliop

cocothebam committed:
update Def

defined on what the code does (move forwords and nbackwords)

cocothebam committed:
fix problems joe found

fix problems joe found

KenwoodFox reviewed (changes requested):
It looks correct! Make sure to resolve the comments on earlier reviews.
Your next step is to call your function from within robot.java, use `joe-drivetrain` as an example.

cocothebam committed:
Quick test see if compiles on other machine

KenwoodFox committed:
pushback

cocothebam committed:
putString added

when turns it says TUUUUUUUUUUUUURRRRRRRRRRRRRRRRRRRRRRNNNNNNNNNNNN
and when it moves it says scurt

cocothebam committed:
Merge branch 'Cohen-drivetrain' of https://github.com/FRC-1721/pre2020season into Cohen-drivetrain

KenwoodFox committed:
hotfix

cocothebam committed:
pushback

cocothebam committed:
invert controls

inverted yaw and inverted control

cocothebam committed:
added command removed subsystem call

added new commands and removed the manual call thought the subsystem

KenwoodFox committed:
Repaired inports

KenwoodFox commented:
You need to test this code first.

cocothebam committed:
Chaging pullstring

Changed pullstring to Turing left and Turning right instead of TUUUUUUUUUUUUUURRRRRRRRRRRRRRRRRRNNNNNN and Scurt

cocothebam committed:
Merge branch 'Cohen-drivetrain' of https://github.com/FRC-1721/pre2020season into Cohen-drivetrain
